server/pbench/bin/pbench-sync-package-tarballs

```diff
@@ -5,7 +5,7 @@
 # WARNING; Do not put anything in here that will pollute the
 # standard output!
 
-export CONFIG=${CONFIG:-/opt/pbench-server/lib/config/pbench-server.cfg}
+export CONFIG=/opt/pbench-server/lib/config/pbench-server.cfg
```

Comment:
Can you add a comment that says, "we ensure we are using the server config file" or something to that effect?